tests/skipped_tests_gpu_no_fp64.tbl

```diff
@@ -123,265 +123,6 @@ tests/third_party/cupy/linalg_tests/test_product.py::TestDotFor0Dim_param_9_{sha
 tests/third_party/cupy/linalg_tests/test_product.py::TestDotFor0Dim_param_10_{shape=((4, 2), ()), trans_a=False, trans_b=True}::test_dot
 tests/third_party/cupy/linalg_tests/test_product.py::TestDotFor0Dim_param_11_{shape=((4, 2), ()), trans_a=False, trans_b=False}::test_dot
 
-tests/third_party/cupy/math_tests/test_arithmetic.py::TestArithmeticUnary_param_29_{arg1=2.0, name='reciprocal'}::test_unary
```

Comment:
can we now enable all math tests in public CI?

Reply:
some of the tests from `third_party/cupy/math_tests/test_sumprod.py` still fail on Windows, all other math tests are now added to public CI.